tests/hpc/mpi_master.pm

```diff
@@ -113,6 +114,10 @@ sub run ($self) {
     }
     barrier_wait('MPI_RUN_TEST');
     record_info 'MPI_RUN_TEST', strftime("\%H:\%M:\%S", localtime);
+
+    record_info 'testing IMB', 'Run all IMB-MPI1 components';
+    my $imb_version = script_output("rpm -q --queryformat '%{VERSION}' imb-gnu-$mpi-hpc");
+    assert_script_run("mpirun -np 4 /usr/lib/hpc/gnu7/$mpi/imb/$imb_version/bin/IMB-MPI1", timeout => 320);
```

Comment:
> There seems to be at least the name of the test missing - like 'PingPong'. I would also recommend to take advantage of the environment modules and use `module load gnu $mpi` instead. I know, openQA doesn't allow to run multiple commands in a single shell, but maybe you could assemble a short script with something similar to a here-document that you load onto the SUT and run it there. Actually, without naming the benchmark to run it may run all. Being that these are benchmarks and we do the tests on VMs without special hardware I'm not so sure how useful it is to run them all - in particular when we are not evaluating the results. This is a test to see if imb works, not a benchmark test at all. For these we use a different testing.

- Yes the 'PingPong' is missing with the intention to run the whole perf tests. i also questioned how useful this might be but they are fast and i can i thought it might give some useful data for comparison. Evaluation can be implemented later.
- it does run `module load gnu $mpi`.
- i dont know about the other benchmark tests

Reply:
Ok, I've looked at the test results, now. IMB seems to pick a subset of all tests. Results look sensible, if the runtime of the test for this subset of tests is not too long, I guess we can stay with that.